Tasks/InstallAppleProvisioningProfileV1/installprovprofile.ts

```diff
@@ -23,15 +23,20 @@ async function run() {
 
             if (tl.filePathSupplied('provProfileSourceRepository') && tl.exist(provProfilePath) && tl.stats(provProfilePath).isFile()) {
                 const info = await sign.installProvisioningProfile(provProfilePath);
-                tl.setTaskVariable('APPLE_PROV_PROFILE_UUID', info.provProfileUUID);
+                if (info && info.provProfileUUID) {
```

Comment:
Should this be
```suggestion
                if (info && info.provProfileUUID && info.provProfileUUID.trim()) {
```
and likewise with the condition on line 36?